modules/javafx.graphics/src/main/java/com/sun/javafx/css/BitSet.java

```diff
@@ -220,19 +224,30 @@ public boolean contains(Object o) {
 
         final T t = cast(o);
 
+        if (t == null) {  // if cast failed, it can't be part of this set
+            return false;
+        }
+
         final int element = getIndex(t) / Long.SIZE;
```

Comment:
`getIndex(t)` is called twice here and below, maybe we should consider calling it once and save the result in a local variable instead?

Reply:
I fixed this.